<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filtdlg</class>
 <widget class="QDialog" name="filtdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>495</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Enable Filtering</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>303</width>
       <height>414</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>303</width>
       <height>414</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>412</height>
       </rect>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>251</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Butterworth low-pass</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Butterworth high-pass</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Butterwoth band-pass</string>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>271</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QDoubleSpinBox" name="freqCut1">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="freqCut2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="freqSlope1">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="freqSlope2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>L.P. frequency cut-off</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Slope (db/oct)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>H.P. frequency cut-off</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Slope (db/oct)</string>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>180</y>
         <width>251</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Band-pass</string>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>281</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QDoubleSpinBox" name="freqCutLow">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="freqPassLow">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="freqPassHigh">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="freqCutHigh">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Low-cut frequency</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Low-pass frequency</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>High-pass frequency</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>140</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>High-cut frequency</string>
        </property>
       </widget>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>freqCut2</tabstop>
  <tabstop>freqSlope2</tabstop>
  <tabstop>freqCut1</tabstop>
  <tabstop>freqSlope1</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
